onedal/covariance/covariance.py

```diff
@@ -37,6 +38,8 @@ def _get_onedal_params(self, dtype=np.float32):
         }
         if daal_check_version((2024, "P", 1)):
             params["bias"] = self.bias
+        if daal_check_version((2024, "P", 400)):
+            params["assumeCentered"] = self.assume_centered
```

Comment:
Related to current PR slightly, more like for future discussion: there are places in onedal4py like this one where no warning is produced if oneDAL version is older than sklearnex and actual behaviour is different from one set by parameters, but this case is expected be rare